### PR TITLE
Move List Bible Books endpoint to /bibles/books.

### PR DESCRIPTION
**Pull request by NateMerritt**

### CODE REVIEW

File: src/Aquifer.Public.API/Endpoints/Bibles/Books/List/Endpoint.cs [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 
-namespace Aquifer.Public.API.Endpoints.BibleBooks.List;
+namespace Aquifer.Public.API.Endpoints.Bibles.Books.List;
 
 public class Endpoint : EndpointWithoutRequest<IEnumerable<Response>>
 {
     public override void Configure()
     {
-        Get("/bible-books");
+        // "/bible-books" is still supported for legacy compatibility but "/bibles/books" is the new hotness
```
Comment (jwinston-bn) [inline src/Aquifer.Public.API/Endpoints/Bibles/Books/List/Endpoint.cs:11]: I think we can remove these comments since the LegacyEndpoint implies it.

**Comment by NateMerritt** [comment]
I updated this PR to use inheritance so that we could have separate docs on the legacy endpoint and the new endpoint in order to have a deprecation message.

File: src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public class LegacyEndpoint : Endpoint
+{
+    public override void Configure()
+    {
+        // "/bible-books" is still supported for legacy compatibility but "/bibles/books" is the new hotness
+        Get("/bible-books");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+
+        // we can eventually hide this endpoint from the docs altogether: https://fast-endpoints.com/docs/swagger-support#filtering-endpoints
```
Comment (jwinston-bn) [inline src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs:13]: Can we just remove it actually?

Comment (NateMerritt) [reply src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs:13]: Sure.  I'll look into apply endpoint labels tomorrow.  I'm hoping we can just make a "no-docs" label or something like that and apply it with a single line of configuration after I wire things up.

File: src/Aquifer.Public.API/Endpoints/Bibles/Books/List/Endpoint.cs (hunk 1 above)
Comment (NateMerritt) [reply src/Aquifer.Public.API/Endpoints/Bibles/Books/List/Endpoint.cs]: Done.

File: src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs (hunk 2 above)
Comment (NateMerritt) [reply src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs:13]: Done [here](https://github.com/BiblioNexusStudio/aquifer-server/pull/445/commits/f6621945b6df3c8565e0404f3648361e715887c9).  I'm pretty happy with this solution.

**Comment by NateMerritt** [comment]
> I updated this PR to use inheritance so that we could have separate docs on the legacy endpoint and the new endpoint in order to have a deprecation message.

Instead of adding the word "deprecated" to the docs this PR now uses Swagger endpoint filtering to omit the deprecated endpoint from the docs entirely:
image

**Review by jwinston-bn — APPROVED** [review]

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**